src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_broodlord_lashlayer.cpp

```diff
@@ -113,7 +122,86 @@ class boss_broodlord : public CreatureScript
     }
 };
 
+class go_suppression_device : public GameObjectScript
+{
+    public:
+        go_suppression_device() : GameObjectScript("go_suppression_device") { }
+
+        struct go_suppression_deviceAI : public GameObjectAI
+        {
+            go_suppression_deviceAI(GameObject* go) : GameObjectAI(go), _instance(go->GetInstanceScript())
+            {
+                _events.ScheduleEvent(EVENT_SUPPRESSION_AURA, Seconds(6));
```

Comment:
I would prefer this to be somewhere else than the constructor

Reply:
thank you :)